gogen/goelements.go

```diff
@@ -644,7 +644,8 @@ func (s *GoLangMapper) yangDefaultValueToGo(value string, args resolveTypeArgs,
 		}
 
 		// Check if the value is an empty string.
-		if len(value) == 0 {
+		signStripedValue := strings.TrimRight(strings.TrimLeft(value, "-"), "+")
```

Comment:
Why is this TrimRight rather than TrimLeft?

Reply:
Oops, thanks for catching it! I guess my auto-completion failed me when I was typing too fast... The tests didn't catch this because of `uint` typos of the YANG types (caused by copy-pasta), and `ParseUint` returned an error anyway because of the sign prefix... I've fixed them in the latest commits.

Reply:
Thanks